docs/root/configuration/listeners/lds.rst

```diff
@@ -18,10 +18,13 @@ The semantics of listener updates are as follows:
 * Listeners are effectively constant once created. Thus, when a listener is updated, an entirely
   new listener is created (with the same listen socket). This listener goes through the same
   warming process described above for a newly added listener.
-* When a listener is updated or removed, the old listener will be placed into a "draining" state
+* When a listener is removed, the old listener will be placed into a "draining" state
   much like when the entire server is drained for restart. Connections owned by the listener will
   be gracefully closed (if possible) for some period of time before the listener is removed and any
   remaining connections are closed. The drain time is set via the :option:`--drain-time-s` option.
+* When a tcp listener is updated, if the new listener contains a subset of filter chains in the old listener,
+  the connections owned by these overlapping filter chains remain open. Only the connections owned by the
+  obsoleted filter chains will be drained following the above pattern. 
```

Comment:
@mattklein123  This is the high level description of the per filter chain update.
Do you want concrete definition of `overlapping`? In this section or somewhere else?

Reply:
I would like to see a section on filter chains here: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/listeners/listeners_toc. You can discuss the high level overview, overlapping, intelligent update, etc. That can resolve https://github.com/envoyproxy/envoy/issues/11088

Reply:
Sure. Will update PR in next commit